@xen-orchestra/web/src/pages/(site)/hosts.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="host" :class="{ mobile: uiStore.isMobile }">
```

Comment:
the class name is not supposed to be the same as the page name?
```suggestion
  <div class="hosts" :class="{ mobile: uiStore.isMobile }">
```